harmony/config.py

```diff
@@ -15,9 +15,10 @@
 
 from dotenv import load_dotenv
 
-Environment = Enum('Environment', ['SBX', 'SIT', 'UAT', 'PROD'])
+Environment = Enum('Environment', ['LOCAL', 'SBX', 'SIT', 'UAT', 'PROD'])
 
 HOSTNAMES = {
+    Environment.LOCAL: 'localhost:3000',
     Environment.SBX: 'harmony.sbx.earthdata.nasa.gov',
```

Comment:
We don't have an external URL for sandbox. We can only tunnel in and hit an internal load balancer.

Reply:
Should I just remove this? I.e., would we potentially expose sandbox in the future?

Reply:
Sandbox accounts aren't allowed to expose any http services. I don't think we need to make harmony-py work for sandbox, so removing anything referencing sandbox makes to me. If we need to make sandbox work that's a new ticket anyway.